### DISCUSSION TITLE
Merge conflicts

### DISCUSSION THREAD

**Discussion by kumawatanupriya** [Project-Books/book-project#649] [Q&A] [answered]
@knjk04 I raised a pull request https://github.com/Project-Books/book-project/pull/647 it shows merge conflict, but when I pull from branch 0.2.0 , then it shows already up to date.

**Comment by knjk04** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/11173328/109493563-993b3c80-7a84-11eb-9d40-01fd56489ad6.png)

It looks like you've incorrectly set the destination branch to `main`. Instead, try setting it to `0.2.0`
